browser/ui/tabs/split_view_browser_data.cc

```diff
@@ -15,16 +15,9 @@
 #include "chrome/browser/ui/tabs/tab_model.h"
 
 SplitViewBrowserData::SplitViewBrowserData(Browser* browser)
```

Comment:
maybe out of scope, but as per @bridiver's [comments](https://bravesoftware.slack.com/archives/C7VLGSR55/p1729368619704789) should we take the chance to update this to pass in a `TabStripModel`

Reply:
We can't remove `Browser` from this argument, as it is used by the base class. Chromium instantiates this class with a template in a generic way. I would have passed `TabStripModel*` here otherwise, but as it is adding an extra `TabStripModel` will just cause noise IMO.